src/sas/qtgui/Utilities/Preferences/PreferencesPanel.py

```diff
@@ -94,6 +98,14 @@ def _saveStagedChanges(self):
         """ When OK or Apply are clicked, all staged changes should be applied to the config. """
         for k, v in self._staged_changes.items():
             setattr(config, k, v)
+        if any(self._staged_requiring_restart):
+            message = "SasView must restart for the following values to take effect:\n"
+            for val in self._staged_requiring_restart:
+                message += f"\t-{val}\n"
+            msgBox = QMessageBox(QMessageBox.Information, "", message, QMessageBox.Ok)
```

Comment:
I'm guessing adding an option for it to restart itself would be a massive pain to get working. I'm not even sure how other software does this.

Reply:
I looked into this, and it's possible, but would add too much scope creep to this PR.

Reply:
cool